extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/StringHelper.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.vault.runtime;
+
+import java.io.UnsupportedEncodingException;
+
+import io.quarkus.vault.VaultException;
+
+public class StringHelper {
+
+    public static final String UTF_8 = "UTF-8";
```

Comment:
You can probably reuse `StandardCharsets`

Reply:
done